Formula/cquery.rb

```diff
@@ -13,10 +13,14 @@ class Cquery < Formula
     sha256 "9692d28682c009a9bec583ff20aa16ce5c790975375755a5a48a28ec3e3953c4" => :el_capitan
   end
 
+  depends_on "cmake" => :build
+  depends_on "llvm"
```

Comment:
Can't this work with system llvm, like it used to?

Reply:
It did not use system llvm. Instead, it used to download a bundled llvm binaries. 

This changes to use the brew llvm instead of bundled one.